### DISCUSSION TITLE
service_code missing in FFS examples but in schema and readme has as required

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#16] [Q&A] [answered]
https://github.com/CMSgov/price-transparency-guide/blob/master/examples/in-network-rates/in-network-rates-fee-for-service-sample.json does not have the service_code field, but schema has the field, and readme has as a required field.  Assumption is that the example is incorrect and schema is accurate???

**Comment by angelamarie0975** [comment]
I have the same question.

**Comment by shaselton-usds** [comment, marked as answer]
Definitely true -- thanks for the catch. Updates here: https://github.com/CMSgov/price-transparency-guide/commit/4b7a1530c462f6272fa2c905f260d7dc4854bb4f

**Reply by taylorpatriciab** [reply]
@shaselton-usds I see the updates for the fee-for-service samples.  Should there be similar updates for the bundle and capitation samples?

**Reply by taylorpatriciab** [reply]
@shaselton-usds my apologies.. I see it in the capitation but I don't see it in the bundle.
